modules/swagger-generator/src/test/java/io/swagger/generator/online/OnlineJavaClientOptionsTest.java

```diff
@@ -0,0 +1,34 @@
+package io.swagger.generator.online;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public class OnlineJavaClientOptionsTest extends OnlineGeneratorOptionsTest {
+
+    public OnlineJavaClientOptionsTest() {
+        super("java", false);
+    }
+
+    protected OnlineJavaClientOptionsTest(String language, boolean isServer) {
+       super(language, isServer);
+    }
+
+    @Override
+    protected Map<String, String> getOptions() {
```

Comment:
@lugaru1234 shouldn't these come directly from loading the `java` client generator and fetching the options programmatically?